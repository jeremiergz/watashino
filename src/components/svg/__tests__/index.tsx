import React from 'react';
import ReactTestRenderer from 'react-test-renderer';
import BookIcon from '../Book';
import BusinessIcon from '../Business';
import DownloadIcon from '../Download';
import HouseIcon from '../House';
import MoodIcon from '../Mood';
import PersonIcon from '../Person';

describe('src/components/svg Test Suite', () => {
    describe('Book Test Suite', () => {
        it('matches snapshot', () => {
            const jsx = <BookIcon />;
            expect(ReactTestRenderer.create(jsx).toJSON()).toMatchSnapshot();
        });
    });

    describe('Business Test Suite', () => {
        it('matches snapshot', () => {
            const jsx = <BusinessIcon />;
            expect(ReactTestRenderer.create(jsx).toJSON()).toMatchSnapshot();
        });
    });

    describe('Download Test Suite', () => {
        it('matches snapshot', () => {
            const jsx = <DownloadIcon />;
            expect(ReactTestRenderer.create(jsx).toJSON()).toMatchSnapshot();
        });
    });

    describe('House Test Suite', () => {
        it('matches snapshot', () => {
            const jsx = <HouseIcon />;
            expect(ReactTestRenderer.create(jsx).toJSON()).toMatchSnapshot();
        });
    });

    describe('Mood Test Suite', () => {
        it('matches snapshot', () => {
            const jsx = <MoodIcon />;
            expect(ReactTestRenderer.create(jsx).toJSON()).toMatchSnapshot();
        });
    });

    describe('Person Test Suite', () => {
        it('matches snapshot', () => {
            const jsx = <PersonIcon />;
            expect(ReactTestRenderer.create(jsx).toJSON()).toMatchSnapshot();
        });
    });
});
