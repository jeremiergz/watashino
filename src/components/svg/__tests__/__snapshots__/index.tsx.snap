// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`components/svg Test Suite Book Test Suite matches snapshot 1`] = `
<svg
  className="sc-AykKC hVlXBK"
  height={32}
  viewBox="0 0 24 24"
  width={32}
  xmlns="http://www.w3.org/2000/svg"
>
  <path
    d="M0 0H24V24H0z"
    fill="none"
  />
  <path
    d="M21 5c-1.11-.35-2.33-.5-3.5-.5-1.95 0-4.05.4-5.5 1.5-1.45-1.1-3.55-1.5-5.5-1.5S2.45 4.9 1 6v14.65c0 .25.25.5.5.5.1 0 .15-.05.25-.05C3.1 20.45 5.05 20 6.5 20c1.95 0 4.05.4 5.5 1.5 1.35-.85 3.8-1.5 5.5-1.5 1.65 0 3.35.3 4.75 1.05.1.05.15.05.25.05.25 0 .5-.25.5-.5V6c-.6-.45-1.25-.75-2-1zm0 13.5c-1.1-.35-2.3-.5-3.5-.5-1.7 0-4.15.65-5.5 1.5V8c1.35-.85 3.8-1.5 5.5-1.5 1.2 0 2.4.15 3.5.5v11.5z"
  />
  <path
    d="M17.5 10.5c.88 0 1.73.09 2.5.26V9.24c-.79-.15-1.64-.24-2.5-.24-1.7 0-3.24.29-4.5.83v1.66c1.13-.64 2.7-.99 4.5-.99zM13 12.49v1.66c1.13-.64 2.7-.99 4.5-.99.88 0 1.73.09 2.5.26V11.9c-.79-.15-1.64-.24-2.5-.24-1.7 0-3.24.3-4.5.83zM17.5 14.33c-1.7 0-3.24.29-4.5.83v1.66c1.13-.64 2.7-.99 4.5-.99.88 0 1.73.09 2.5.26v-1.52c-.79-.16-1.64-.24-2.5-.24z"
  />
</svg>
`;

exports[`components/svg Test Suite Business Test Suite matches snapshot 1`] = `
<svg
  className="sc-AykKC hVlXBK"
  height={32}
  viewBox="0 0 24 24"
  width={32}
  xmlns="http://www.w3.org/2000/svg"
>
  <path
    d="M0 0h24v24H0zm10 5h4v2h-4zm0 0h4v2h-4z"
    fill="none"
  />
  <path
    d="M10 16v-1H3.01L3 19c0 1.11.89 2 2 2h14c1.11 0 2-.89 2-2v-4h-7v1h-4zm10-9h-4.01V5l-2-2h-4l-2 2v2H4c-1.1 0-2 .9-2 2v3c0 1.11.89 2 2 2h6v-2h4v2h6c1.1 0 2-.9 2-2V9c0-1.1-.9-2-2-2zm-6 0h-4V5h4v2z"
  />
</svg>
`;

exports[`components/svg Test Suite Download Test Suite matches snapshot 1`] = `
<svg
  className="sc-AykKC hVlXBK"
  height={32}
  viewBox="0 0 24 24"
  width={32}
  xmlns="http://www.w3.org/2000/svg"
>
  <path
    d="M19 9h-4V3H9v6H5l7 7 7-7zM5 18v2h14v-2H5z"
  />
  <path
    d="M0 0h24v24H0z"
    fill="none"
  />
</svg>
`;

exports[`components/svg Test Suite House Test Suite matches snapshot 1`] = `
<svg
  className="sc-AykKC hVlXBK"
  height={32}
  viewBox="0 0 24 24"
  width={32}
  xmlns="http://www.w3.org/2000/svg"
>
  <path
    d="M0 0H24V24H0z"
    fill="none"
  />
  <path
    d="M19 9.3V4h-3v2.6L12 3 2 12h3v8h5v-6h4v6h5v-8h3l-3-2.7zm-9 .7c0-1.1.9-2 2-2s2 .9 2 2h-4z"
  />
</svg>
`;

exports[`components/svg Test Suite Mood Test Suite matches snapshot 1`] = `
<svg
  className="sc-AykKC hVlXBK"
  height={32}
  viewBox="0 0 24 24"
  width={32}
  xmlns="http://www.w3.org/2000/svg"
>
  <path
    d="M0 0h24v24H0z"
    fill="none"
  />
  <path
    d="M11.99 2C6.47 2 2 6.48 2 12s4.47 10 9.99 10C17.52 22 22 17.52 22 12S17.52 2 11.99 2zM12 20c-4.42 0-8-3.58-8-8s3.58-8 8-8 8 3.58 8 8-3.58 8-8 8zm3.5-9c.83 0 1.5-.67 1.5-1.5S16.33 8 15.5 8 14 8.67 14 9.5s.67 1.5 1.5 1.5zm-7 0c.83 0 1.5-.67 1.5-1.5S9.33 8 8.5 8 7 8.67 7 9.5 7.67 11 8.5 11zm3.5 6.5c2.33 0 4.31-1.46 5.11-3.5H6.89c.8 2.04 2.78 3.5 5.11 3.5z"
  />
</svg>
`;

exports[`components/svg Test Suite Person Test Suite matches snapshot 1`] = `
<svg
  className="sc-AykKC hVlXBK"
  height={32}
  viewBox="0 0 24 24"
  width={32}
  xmlns="http://www.w3.org/2000/svg"
>
  <path
    d="M12 12c2.21 0 4-1.79 4-4s-1.79-4-4-4-4 1.79-4 4 1.79 4 4 4zm0 2c-2.67 0-8 1.34-8 4v2h16v-2c0-2.66-5.33-4-8-4z"
  />
  <path
    d="M0 0h24v24H0z"
    fill="none"
  />
</svg>
`;
